user/conditional-builds-stages-jobs.md

```diff
@@ -46,10 +50,44 @@ jobs:
 ```
 {: data-file=".travis.yml"}
 
-Jobs need to be listed explicitly, i.e., using `jobs.include` (or its alias `matrix.include`), in order to specify conditions for them. Jobs created via [matrix expansion](/user/customizing-the-build/#build-matrix) currently cannot have conditions.
+Jobs need to be listed explicitly, i.e., using `jobs.include` (or its alias `matrix.include`), in order to specify conditions for them. Jobs created via [matrix expansion](/user/customizing-the-build/#build-matrix) currently cannot have conditions, but they can be conditionally excluded (see the next section).
```

Comment:
"See below", or an anchor is preferable.

Reply:
done